pyfarm/agent/http/api/assign.py

```diff
@@ -187,15 +187,15 @@ def assignment_started(_, assign_id):
 
         def assignment_stopped(_, assign_id):
             logger.debug("Assignment %s has stopped", assign_id)
+            if (len(config["current_assignments"]) <= 1 and
```

Comment:
Shouldn't this be `< 1` or `== 0`?  If `current_assignments` is 1 or higher I'd think the agent would be running.